adcm_client/objects.py

```diff
@@ -650,6 +668,9 @@ class Action(BaseAPIObject):
     subs = None
     config = None
 
+    def __repr__(self):
+        return f"<Action {self.name} at {id(self)}>"
```

Comment:
There is no info about parent object? Cluster, service, or host

Reply:
There is pretty complicated object hierarchy for Action, Task and Job objects. According to existing class code there is no simple way to get parent object.